CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ option(INPUTLEAP_USE_EXTERNAL_GTEST "Use external installation of Google Test fr
 option(INPUTLEAP_BUILD_X11 "Build with XWindows support" ON)
 option(INPUTLEAP_BUILD_LIBEI "Build with libei support" OFF)
 option(INPUTLEAP_DEPLOY_FLATPAK_SCRIPT "Deploy Flatpak script [Experimental]" OFF)
+
+include (cmake/Version.cmake)
```

Comment:
Please make spacing consistent at least in lines that this PR touches.

Reply:
Fixed in #1946 (that should land first since its this items base